.travis/install.sh

```diff
@@ -36,6 +36,9 @@ if [[ "$DARWIN" = true ]]; then
             curl -O https://bootstrap.pypa.io/get-pip.py
             sudo python get-pip.py
             ;;
+        bundle)
```

Comment:
I've add a travis build step that builds using the system python. This way both options are available. 
I do agree that building use the system python is actually easier and still presents a stable target.

I've added steps to the getting_started docs detailing how to build the application using either the system python (semi-standalone) or using a brew python (standalone).